clarifying in-place updates

mairaw opened the pull request:
Adding some info to the topic since we got a LiveFyre question on this

@rpetrusha, please review to make sure I captured the info correctly

mairaw committed:
clarifying in-place updates

rpetrusha commented on docs/framework/get-started/system-requirements.md:
```diff
@@ -26,6 +22,8 @@ manager: "wpickett"
 
 The tables in this topic provide the hardware, operating system, and software requirements for the .NET Framework 4.5 and its point releases (4.5.1 and 4.5.2), the [!INCLUDE[net_v46](../../../includes/net-v46-md.md)] and its point releases (4.6.1 and 4.6.2), and the .NET Framework 4.7. Development environments that enable you to develop apps for the .NET Framework have a separate set of requireme ...
 
+All .NET Framework versions since .NET Framework 4 are in-place updates. In other words, if there's a later version installed on the machine already, you don't have to install the previous versions. Similarly, if you install a later version, you don't have to first uninstall the previous versions. And, if an OS already comes pre-installed with a particular .NET Framework version, you cannot insta ...
```
This is worth modifying somewhat and doing as a note; I don't remember, though, if notes support bulleted lists, and if Important is a note category. The text that I'd change it do is:
All .NET Framework versions since .NET Framework 4 are in-place updates; only a single 4.x version can be present on a system. In addition, particular versions of the .NET Framework are pre-installed on some versions of the Windows operating system. This means that:
- If there's a later version installed on the machine already, you cannot install a previous version.
- If the OS comes pre-installed with a particular .NET version, you cannot install a previous 4.x version on the same machine.
- If you install a later version, you don't have to first uninstall the previous version.

It probably would be good to make the same change to the Install the .NET Framework for developers topic; too many customers seem to overlook the comment. I can open a separate PR for that tomorrow (and also incorporate the other changes in the version table).

mairaw replied:
Sounds good. I'll give it a try and even I was making that confusion. so adding this note I think it will help. And yes, there is an important note.
I'll add the text in an include file so it can be easily added to topics and we'll see if it supports bullet lists. I'm not sure.

rpetrusha reviewed (changes requested):
@mairaw, I suggested some rewording, as well as a note rather than just a paragraph of text.

rpetrusha commented:
Thanks, @mairaw. The include file is a great idea.

mairaw committed:
Merge branch 'master' into mairaw-patch-10

mairaw committed:
feedback

mairaw committed:
removed old content

mairaw committed:
added description

mairaw commented:
Ok @rpetrusha. I finally addressed your feedback here. I looked at the install guide and saw that this info is there with more detailed examples. Do you really wanna pull that content out and add just the note?

rpetrusha commented:
This looks really good now, @mairaw. It's good to merge when you're ready and the build completes.
In terms of the Install the .NET Framework for Developers topic, I don't think that we want to pull out the content and replace it with a note. It could use some reformatting, though, for clarity; I'll open a PR to do that next week.

mairaw commented:
Sounds good @rpetrusha. Thanks for your review!

mairaw commented:
@rpetrusha do you need to open an issue for the install work item?

rpetrusha commented:
I just opened it, @mairaw. Issue #4318.
